crates/pathfinder/examples/estimate_past_transactions.rs

```diff
@@ -0,0 +1,341 @@
+use anyhow::Context;
+use stark_hash::StarkHash;
+// use web3::types::H256;
```

Comment:
```suggestion
```

Reply:
I hate the uses anyways, dunno how could I had forgotten this :D